### PR TITLE
NAC-130: Admin Console fulltext reindex

### PR DESCRIPTION
**Pull request by Nishant0928**
https://hyland.atlassian.net/browse/NAC-147
https://hyland.atlassian.net/browse/NAC-145
https://hyland.atlassian.net/browse/NAC-130
https://hyland.atlassian.net/browse/NAC-136

### CODE REVIEW

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts [hunk 1]
```diff
@@ -30,6 +30,8 @@ import { ErrorModalComponent } from "../error-modal/error-modal.component";
 import { featureMap, FEATURES } from "../../generic-multi-feature-layout.mapping";
 import { PICTURE_RENDITIONS_LABELS } from "../../../../pictures/pictures-renditions.constants";
 import { THUMBNAIL_GENERATION_LABELS } from "../../../../thumbnail-generation/thumbnail-generation.constants";
+import { FULLTEXT_REINDEX_LABELS } from "src/app/features/fulltext-reindex/fulltext-reindex.constants";
```
Comment (swarnadipa-dev) [inline nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts:33]: Use relative path for import like ../../

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.ts [hunk 2]
```diff
@@ -83,6 +85,7 @@ export class DocumentTabComponent implements OnInit, OnDestroy {
   ) {
     this.inputForm = this.fb.group({
       inputIdentifier: ["", Validators.required],
+      force: [false],
```
Comment (swarnadipa-dev) [inline nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.ts:88]: Do we need this ? as it is applicable only for fulltext & not other features

**Review by swarnadipa-dev — CHANGES_REQUESTED** [review]

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.scss [hunk 3]
```diff
@@ -40,9 +40,29 @@
...
     margin-top: -15px;
   }
+  .force-checkbox-container {
+    display: flex;
+    align-items: center;
+    margin: 0 0 10px;
+  }
+
+  .force-conversion-helper-text {
+    max-width: 600px; /* Adjust as needed */
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.scss:59]: remove comments

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts [hunk 4]
```diff
@@ -30,6 +30,8 @@ import { ErrorModalComponent } from "../error-modal/error-modal.component";
...
+import { MatCheckboxModule } from "@angular/material/checkbox";
```
Comment (madhurkulshrestha-hyland) [inline nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts:34]: unused variable

**Review by madhurkulshrestha-hyland — CHANGES_REQUESTED** [review]

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts (hunk 1 above)
Comment (Nishant0928) [reply nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts]: done

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.scss (hunk 3 above)
Comment (Nishant0928) [reply nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.scss]: done

File: nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts (hunk 4 above)
Comment (Nishant0928) [reply nuxeo-admin-console-web/angular-app/src/app/features/sub-features/generic-multi-feature-layout/components/document-tab/document-tab.component.spec.ts]: removed the unused variable

**Review by rakeshkumar1019 — APPROVED** [review]

**Review by madhurkulshrestha-hyland — APPROVED** [review]

**Review by swarnadipa-dev — APPROVED** [review]

**Pull request merged**